workflow/rules/quantify.smk

```diff
@@ -468,35 +496,35 @@ rule quant_mirna_pri:
 
 
 rule merge_tables:
-   input:
-       table=expand(
-           os.path.join(
-               config["output_dir"], "TABLES", "{mir}_counts_{sample}"
-           ),
-           sample=pd.unique(samples_table.index.values),
-           mir=config["mir_list"],
-       ),
-       script=os.path.join(config["scripts_dir"], "merge_tables.R"),
-   output:
-       table=os.path.join(config["output_dir"], "TABLES", "counts.{mir}.tab"),
-   params:
-       cluster_log=os.path.join(
-           config["cluster_log"], "merge_tables_{mir}.log"
-       ),
-       prefix="{mir}_counts_",
-       input_dir=lambda wildcards, input: Path(input[0]).parent,
-   log:
-       os.path.join(config["local_log"], "merge_tables_{mir}.log"),
-   container:
-       "docker://zavolab/r-tidyverse:3.5.3"
-   shell:
-       "(Rscript \
-       {input.script} \
-       --input_dir {params.input_dir} \
-       --output_file {output.table} \
-       --prefix {params.prefix} \
-       --verbose \
-       ) &> {log}"
+    input:
+        table=expand(
+            os.path.join(
+                config["output_dir"], "TABLES", "{mir}_counts_{sample}"
+            ),
+            sample=pd.unique(samples_table.index.values),
+            mir=[mir for mir in config["mir_list"] if mir != 'isomir'],
```

Comment:
In order to avoid hardcoding it I tried using [specific wildcards constrains](https://snakemake.readthedocs.io/en/stable/tutorial/additional_features.html#constraining-wildcards) as well as the [general wildcards constrains](https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#wildcards). None of them worked. I can either try to find another way now or maybe open an issue for this (?)

Reply:
No, as I said, just leave it :)